packages/client/src/views/RecordAudit/ActionMenu.tsx

```diff
@@ -136,10 +133,11 @@ export const ActionMenu: React.FC<{
   return (
     <>
       <DropdownMenu id="action">
-        <DropdownMenu.Trigger asChild>
-          <PrimaryButton icon={() => <CaretDown />}>
+        <DropdownMenu.Trigger>
```

Comment:
```suggestion
        <DropdownMenu.Trigger asChild>
```
We need the `asChild` prop here for proper DOM element hierarchy